tools/kafka_sasl_ssl/generate_ssl_CA/ssl_one_way.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# The example: bash ssl_one_way.sh kafka-0.tigergraph.com ~/SSL_one_way tiger123
+# The example password: tiger123
+# The example server host name: kafka-0.tigergraph.com
+# The certificate generation path: ~/SSL_one_way
+
+if [ $# -eq 3 ]; then
+  server_hostname=$1
+  generate_root=$2
+  pass=$3
+else
+  echo "Error in parameter. Please check."
+  echo "e.g. bash ssl_one_way.sh server_hostname generate_root password"
```

Comment:
please change it to options, add allow default values, if not given, use the default value. You may learn from our installer:
~/product/src/er/offline_install/installer/normal_install/install.sh